.release-please-manifest.json

```diff
@@ -15,5 +15,6 @@
   "packages/sdk-node": "22.2.0",
   "packages/sdk-rtl": "21.3.1",
   "packages/wholly-sheet": "0.5.25",
-  "python": "22.2.1"
+  "python": "22.2.1",
+  "go": "0.0.1"
```

Comment:
hm, IIRC this actually won't do anything. The way release-please works is it looks at the contents of this file at the commit that represents the last release PR merging into `main`, not what's actually at HEAD.

I can't remember what the changelog behavior looks like when adding a new package. The default version gets set [like this](https://github.com/googleapis/release-please/blob/cb0f936e598c5bfb8d3e0a9c0eeb57ae59a35140/src/strategies/base.ts#L501) BUT you'll want to over-ride it with a `"release-as": "22.4.0"` tag (either directly the go config in release-please-config.json or inheriting it from top level if you get into the next release)

Reply:
https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#manifest
```
[](https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#using-with-github-actions-and-lerna)Once the first release PR has been merged, subsequent release-pr runs will retrieve the content of the manifest at the commit corresponding to that merged PR. It will use this to find the last-released version for each package. It will only read the manifest versions at the tip of the default/configured branch if it fails to find a package's version in the last released manifest content. It will only consider version info it is missing for a configured package (thus handling the new package bootstrap case).
```

Reply:
Mot importantly ` It will only read the manifest versions at the tip of the default/configured branch if it fails to find a package's version in the last released manifest content.`.

Reply:
So we should be good. 👍

Reply:
ah, ok.